1 table.sql

```diff
@@ -0,0 +1,68 @@
+    PassportIssueDate   date,
+    PassportExpiryDate  date,
+    Nationality         varchar (30),               constraint ck_Nationality_may_only_contain_letters check(Nationality not like '%[^A-Z]%'), 
+    CheckedIn as case when PassportNumber > '' then 1 else 0 end persisted,
+                                                        constraint ck_Date_arriving_cannot_be_before_date_departing check (DateDeparting <= DateArriving),
+                                                        constraint ck_Passenger_must_be_between_the_age_of_16_and_90 check (datediff(day, DOB , DateDeparting ) >= 5844 and datediff(day, DOB , DateDeparting ) < 32873),
+                                                        constraint ck_Passport_issue_date_cannot_be_more_then_5_years_ago_if_passenger_is_under_16_or_more_then_10_years_for_all_other_passengers 
+                                                                    check (  ((datediff(day, DOB , DateDeparting ) <= 5844) and (datediff(day, PassportIssueDate, DateDeparting) <= 1825)) 
+                                                                        or ((datediff(day, DOB , DateDeparting ) > 5844) and (datediff(day, PassportIssueDate, DateDeparting) <= 3652))  ),
+                                                        constraint ck_Passport_expiry_date_cannot_be_before_Issue_Date check (PassportExpiryDate > PassportIssueDate),
+                                                        constraint ck_If_flight_is_international_and_arrival_country_is_not_passports_country_passport_must_be_valid_for_at_least_another_6_month 
+                                                                    check 
+                                                                    (   ((ArrivalCountry = DepartureCountry) or (ArrivalCountry = Nationality)) 
+                                                                    or ((ArrivalCountry <> DepartureCountry) and (ArrivalCountry <> Nationality) and (datediff(day, DateArriving, PassportExpiryDate) > 180)) )
+                                                                    ,
+                                                        constraint ck_Passport_must_be_valid_upon_entry check (PassportExpiryDate >= DateArriving),
+                                                        constraint u_Only_one_passenger_per_flight_allowed unique (FlightNumber, DateDeparting, PassengerFirstName, PassengerLastName ),
+                                                        constraint ck_Booking_maximum_one_year_in_advance check (datediff(day, getdate(), DateDeparting ) <= 365),
+                                                        constraint ck_Check_in_can_be_done_earliest_30_days_before_depature check ( (PassportNumber = '' or (PassportNumber <> '' and datediff(day, getdate(), DateDeparting ) <= 30)) )
+                                                        
```

Comment:
It would be nice to check that DepartureAirport is not the same as ArrivalAirport

Reply:
fixed it